frontend/src/components/App/runCommand.ts

```diff
@@ -37,7 +37,7 @@ export function runCommand(
     throw new Error('runCommand only works in Headlamp app mode.');
   }
 
-  if (process.env.REACT_APP_ENABLE_RUN_CMD !== 'true') {
+  if (import.meta.env.REACT_APP_ENABLE_RUN_CMD !== 'true') {
```

Comment:
Can support for process.env be kept somehow?

Mostly interested for a future move of the plugins to vite.

Reply:
revisited this topic. I removed the process env plugin because app itself is using import.meta.env and plugins don't have access to this since all the env variables are statically replaced during build time and process.env is not available at all. 
you can check it by printing process in the dev console in the current version of the app

Reply:
We have an issue about how plugins cannot access the process.env anyway, or maybe that's an opportunity for not having the process anymore, if plugins will be able to access the vars differently anyway.

Reply:
It would be better to keep process.env because it's a known node API, and also known in other tools like in vue and create-react-app / nextjs / etc.

Reply:
aliasing import.meta.env to process.env breaks the process polyfill so it's one or the other.
I've tried to keep process.env but it just doesn't work together with polyfill unfortunately.

in addition import.meta is part of the standard [import.meta - JavaScript | MDN (mozilla.org)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import.meta) unlike process which is a node specific thing that we should try to move away from (similar to commonjs)